### DISCUSSION TITLE
Error when using wait with expected condition

### DISCUSSION THREAD

**Discussion by DudeParaK** [ElSnoMan/pyleniumio#215] [Q&A] [answered]
Imported the method 
from selenium.webdriver.support import expected_conditions as ec
and tried using it
Getting the below error 
py.wait().until(ec.element_to_be_clickable(By.XPATH,"//div[@id='bd-menu']"))
E       TypeError: __init__() takes 2 positional arguments but 3 were given

Could you please help me to find what is wrong with the statement

**Comment by vernko** [comment, marked as answer]
Pylenium already had waiting built in. It does this automatically.

I’d check this out first and use PyleniumWait that is built in too,
https://docs.pylenium.io/pylenium-commands/wait
